po/common/common.pot

```diff
@@ -7452,5 +7452,5 @@ msgid "Available on F-Droid"
 msgstr "Available on F-Droid"
 
 msgctxt "update_nutrients_data"
-msgid "Update nutrients data"
-msgstr "Update nutrients data"
+msgid "Extract Nutrition facts"
```

Comment:
For consistency, we never capitalize words in titles.

```suggestion
msgid "Extract nutrition facts"
```

(same for 3 others)